marconi-chain-index/json-rpc/src/Network/JsonRpc/Types.hs

```diff
@@ -111,18 +116,33 @@ data JsonRpcErr e = JsonRpcErr
 parseErrorCode :: Int
 parseErrorCode = -32700
 
+parseErrorMessage :: String
+parseErrorMessage = "Parse error"
```

Comment:
Don't we want to have a datatype that put together the error and the message?

Reply:
Will do that.

Reply:
Can you validate the change? 43cdc8197e76de8d09ed49117fe71523ad821801

Reply:
Even better.